airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_incremental.py

```diff
@@ -180,24 +245,35 @@ def test_two_sequential_reads(
             latest_state = states_1[-1].state.data
             state_input = states_1[-1].state.data
 
-        parsed_records_1 = list(records_with_state(records_1, latest_state, stream_mapping, cursor_paths))
-
-        # This catches the case of a connector that emits an invalid state that is not compatible with the schema
-        # See https://github.com/airbytehq/airbyte/issues/21863 to understand more
-        assert parsed_records_1, "At least one valid state should be produced, given a cursor path"
-
-        for record_value, state_value, stream_name in parsed_records_1:
```

Comment:
Copied from PR description:
---
**2. State is not meant to be readable**
We have been treating state as parseable.

We should never do this.

The connector sets and interprets its own state.

There are many cases where us as a platform will never understand the contents of a state message